src/core-messages.lisp

```diff
@@ -681,6 +609,78 @@
   :documentation "Specify an acquisition on an rx-frame as well as the\
       filters to apply.")
 
+(defmessage |Program| ()
+    (
+     (|waveforms|
+      :documentation "The waveforms appearing in the program by waveform\
+          label."
+      :type (:map :string -> |AbstractWaveform|)
+      :required t
+      :default nil)
+
+     (|filters|
+      :documentation "The readout filter kernels appearing in the program by\
+          feature label."
+      :type (:map :string -> |AbstractKernel|)
+      :required t
+      :default nil)
+
+     (|scheduled_instructions|
+      :documentation "The ordered sequence scheduled instruction objects."
+      :type (:list |Instruction|)
```

Comment:
This is the only line of actual diff. The `:type` here changed from `(:list :map)` to `(:list |Instruction|)`, but as a result these messages needed to be moved after the definition of the `|Instruction|` message, otherwise you get an error when attempting to import `core_messages.py`.